crates/gpui/src/text_system/font_features.rs

```diff
@@ -142,3 +142,19 @@ impl schemars::JsonSchema for FontFeatures {
         schema.into()
     }
 }
+
+fn is_valid_feature_tag(tag: &str) -> bool {
+    let raw_bytes = tag.bytes();
+    if raw_bytes.len() != 4 {
+        return false;
+    }
+    for ch in raw_bytes.into_iter() {
+        match ch {
+            48..=57 => continue,  // 0 ... 9
```

Comment:
Seems like we could operate on `char`s instead to make these match arms a bit more self-explanatory?

```suggestion
            '0'..='9' => continue,  // 0 ... 9
```

Reply:
Thanks! Changes updated.